platform/ios/config.cmake

```diff
@@ -1,5 +1,10 @@
 add_definitions(-DMBGL_USE_GLES2=1)
 
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -flto=thin")
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -flto=thin")
+set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} -flto=thin")
+set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "${CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO} -flto=thin")
```

Comment:
I should also note that this config doesn’t affect the mbgl targets in the macos project, so those currently don’t have LTO set.

Reply:
Yeah, ideally we'd set the Xcode configuration settings. However, technically you could also run `cmake .. -GNinja` on macOS to build on macOS without Xcode.